core/sql/sqlcomp/CmpSeabaseDDLtable.cpp

```diff
@@ -8358,6 +8358,19 @@ desc_struct * CmpSeabaseDDL::getSeabaseUserTableDesc(const NAString &catName,
       Lng32 keyColCount = *(Lng32*)vi->get(6);
       Lng32 nonKeyColCount = *(Lng32*)vi->get(7);
       Lng32 idxNumSaltPartns = *(Lng32*)vi->get(8);
+      char * format = vi->get(9);
+      Lng32 idxRowFormat;
+
+      if (memcmp(format, COM_ALIGNED_FORMAT_LIT, 2) == 0)
```

Comment:
Not sure I understand this: This gets the row format from the TABLES table, so the index row format is the same for all the indexes, is that correct? Wouldn't we need to do that per index? For example, older tables could have an aligned base table and unaligned indexes, while newer tables and indexes could both be aligned?